litellm/llms/anthropic/chat/handler.py

```diff
@@ -318,6 +325,37 @@ def _process_response(
         model_response._hidden_params = _hidden_params
         return model_response
 
+    @staticmethod
+    def _convert_tool_response_to_message(
```

Comment:
@ishaan-jaff this code should **not** be in the handler function as it's about llm translation.

it should be in the transformation.py file 

---
acknowledging prior code was in handler - it seems like that was missed during a refactor.

Reply:
hey I think a lot of the previous code for this was written by you @krrishdholakia

Reply:
This is the PR I looked at - had no idea why this logic was in the handler to begin with but assumed there must have been a reason you had https://github.com/BerriAI/litellm/commit/f2401d6d5eadef39036feacb034dba6f5e102a74#diff-4b817e67059fdef6b8fabcd1e356d6597ffdde6ff251a1233a01d41b8805207b

Reply:
fixed here @krrishdholakia https://github.com/BerriAI/litellm/pull/6834